environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java

```diff
@@ -644,6 +671,8 @@ public Weld reset() {
         enabledInterceptors.clear();
         enabledDecorators.clear();
         extensions.clear();
+        containerLifecycleObservers.clear();
```

Comment:
Was this an oversight from previous changes? Can't see any relation to this issue.